cmds/core/mount/mount.go

```diff
@@ -85,45 +108,43 @@ func getSupportedFilesystem(originFS string) ([]string, bool, error) {
 	return returnValue, known, err
 }
 
-func informIfUnknownFS(originFS string) {
-	knownFS, known, err := getSupportedFilesystem(originFS)
+func (c *cmd) informIfUnknownFS(originFS string) {
+	knownFS, known, err := c.getSupportedFilesystem(originFS)
 	if err != nil {
 		// just don't make things even worse...
 		return
 	}
 	if !known {
-		log.Printf("Hint: unknown filesystem %s. Known are: %v", originFS, knownFS)
+		fmt.Fprintf(c.stderr, "Hint: unknown filesystem %s. Known are: %v", originFS, knownFS)
 	}
 }
 
-func main() {
-	if len(os.Args) == 1 {
-		n := []string{"/proc/self/mounts", "/proc/mounts", "/etc/mtab"}
-		for _, p := range n {
+func (c *cmd) run(args ...string) error {
+	if len(args) == 0 {
+		for _, p := range c.mountsPath {
 			if b, err := os.ReadFile(p); err == nil {
-				fmt.Print(string(b))
-				os.Exit(0)
+				c.stdout.Write(b)
+				return nil
 			}
 		}
-		log.Fatalf("Could not read %s to get namespace", n)
+		return fmt.Errorf("%w: %v to get namespace", errMountPath, c.mountsPath)
```

Comment:
I've been told by other reviewers that the error should come last, wdyt?
"%q:%w", c.mountsPath, errMountPath)
although tbh in this case I think you did the right thing ...

Reply:
I think it true, but here I use %w to simplify testing and for nothing else.